### PR TITLE
Add cfn

### LINKED ISSUE #136: CloudFormation support
[CloudFormation Linter](https://github.com/aws-cloudformation/cfn-python-lint/)

[CloudFormation template recognition logic](https://github.com/aws-cloudformation/aws-cfn-lint-visual-studio-code/blob/ec7ab4af616cf85c18f93918cdeeaa11da2fbe72/server/src/server.ts#L140) if needed

### PR DESCRIPTION
**Pull request by admiralAwkbar**
Resolves #136

This kind of files are "specials" because the extensions json/yaml are already checked by specifics linters. I've bypassed checks & test in the script, but i still have to handle the print of header `Linting [...] files` for a bypassed type:

```
----------------------------------------------
----------------------------------------------
Linting [JSON] files...
----------------------------------------------
----------------------------------------------

----------------------------------------------
----------------------------------------------
Linting [CFN] files...
----------------------------------------------
----------------------------------------------
---------------------------
File:[./file.json]
ERROR! Found errors in [cfn-lint] linter!
ERROR:[E3001 Invalid or unsupported Type AWS::Route53::HostedZonee for resource DNS in us-east-1
./file.json:6:10]

----------------------------------------------
----------------------------------------------
The script has completed
----------------------------------------------
----------------------------------------------

ERRORS FOUND in CFN:[1]
Exiting with errors found!
```

EDIT: just notice that https://github.com/github/super-linter/wiki/Adding-new-language-support list all needed changes, i'll update the PR.

Closes #231

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ Developers on **GitHub** can call the **GitHub Action** to lint their code base
 | **Kotlin**       | [ktlint](https://github.com/pinterest/ktlint)                            |
 | **OpenAPI**      | [spectral](https://github.com/stoplightio/spectral)                      |
 | **Protocol Buffers** | [protolint](https://github.com/yoheimuta/protolint)                  |
+| **AWS CloudFormation templates** | [cfn-lint](https://github.com/aws-cloudformation/cfn-python-lint/)       |
```
Comment (zkoppert) [inline README.md:60]: Can we get this list back in alphabetical order?

Comment (admiralAwkbar) [reply README.md]: yessir...

File: lib/linter.sh [hunk 2]
```diff
@@ -442,6 +450,57 @@ DetectOpenAPIFile()
...
+  ################################
+  if shyaml values-0 Resources | grep -q -E "Type: (AWS|Alexa|Custom)" < "$FILE"; then
+    # Found it
+    return 0
+  fi
+
+  ##########################################
+  # No identifiers of a CFN template found #
+  ##########################################
+  return 1
```
Comment (zkoppert) [inline lib/linter.sh:502]: Should we print an error message noting that no id of a CFN was found?

File: lib/linter.sh [hunk 3]
```diff
@@ -736,7 +796,6 @@ if [[ "$TEST_CASE_RUN" != "false" ]]; then
   # Code will exit from inside this loop
   RunTestCases
 fi
-
```
Comment (zkoppert) [inline lib/linter.sh:739]: Probably want to keep this space

File: lib/linter.sh (hunk 2 above)
Comment (admiralAwkbar) [reply lib/linter.sh:502]: As its looking over yml and json files to see if they could also be cloud formation, we want to be quiet on it, or else we would have a lot of false positive messages

File: README.md (hunk 1 above)
Comment (admiralAwkbar) [reply README.md]: done :)

File: lib/linter.sh (hunk 3 above)
Comment (admiralAwkbar) [reply lib/linter.sh]: fixed all the spacing issues I seemed to have created.. my bad...

**Pull request merged**